### ISSUE TITLE
Safari: Local files don't work with the extension

### ISSUE THREAD

**Issue by monzug** [alpheios-project/webextension#185]
Safari only, tested in build 2.1.0.17
The extension doesn't work with local files loaded in the browser in Safari. 
Open the text file with all the supported languages. Turn on Alpheios. the red icon is on but info panel is not generated and double clicking is not working. also the context menu does not have any reference to Alpheios tools.

**Comment by monzug** [comment]
still doesn't work - tested in Safari App Extension 3.0.2.86
Alpheios Components 1.4.2

**Comment by balmas** [comment]
I don't think we can really do anything about this unfortunately. It's a limitation of the Safari app extension technology.
